packages/engine/tests/units/multiple_groups/ageing-agents/src/init.js

```diff
@@ -0,0 +1,11 @@
+const init = (context) => {
+  var agents = [];
+  for (var i = 0; i < context.globals().num_agents; ++i) {
+    agents.push({
+      agent_id: "00000000-0000-0000-0000-" + String(i + 1).padStart(12, "0"),
```

Comment:
It's not important to change this now, but maybe we could avoid manually writing the UUID format, by using `agent_name` instead of `agent_id` or by using a function in `hash_stdlib` for generating agent ids.

Reply:
Actually, I only used this for testing purposes to see the actual agent id. I'll just remove the id - it's not used.